### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 24

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#200] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254202-d9df5936-e463-4f4c-9b87-ae892b688bbd.png)

**Comment by Exyss** [comment]
Riporto la mia soluzione riportata già sui miei appunti (per chi fosse interessato, si trovano [qui](https://github.com/Exyss/university-notes))

![Esercizio 24](https://user-images.githubusercontent.com/65043207/152653914-11c32bdf-1df8-4c6a-9247-b98a541a226d.png)

**Reply by DavideGalilei** [reply]
ATTENZIONE: c'è un errore nei segni per il PIE di $\large{\vert A \cup B \cup C \cup D \vert}$ (sono stati scambiati nell'espressione finale). L'espressione corretta è:
$\large{\vert A \cup B \cup C \cup D \vert = 4 \cdot \binom{9}{3} - 6 \cdot \binom{5}{3} + 4 \cdot 0 - 1 \cdot 0 = 336 - 60 + 0 - 0 = 276}$

La soluzione poi diventerebbe:
$\text{Soluzione} = \text{Totale} - \large{\vert A \cup B \cup C \cup D \vert} =$
$\large{\binom{10 + 4 - 1}{4 - 1} - 276 = \boxed{10}}$
